examples/04_glm_first_level/plot_localizer_surface_analysis.py

```diff
@@ -184,24 +175,21 @@
 
 fsaverage_data = load_fsaverage_data(data_type="sulcal")
 
-for index, (contrast_id, contrast_val) in enumerate(contrasts.items()):
+for contrast_id, contrast_val in contrasts.items():
     # compute contrast-related statistics
     z_score = glm.compute_contrast(contrast_val, stat_type="t")
 
     # we plot it on the surface, on the inflated fsaverage mesh,
     # together with a suitable background to give an impression
     # of the cortex folding.
     for hemi in ["left", "right"]:
-        print(
-            f"  Contrast {index + 1:1} out of {len(contrasts)}: "
-            f"{contrast_id}, {hemi} hemisphere"
-        )
         plot_surf_stat_map(
             surf_mesh=fsaverage5["inflated"],
             stat_map=z_score,
             hemi=hemi,
             title=contrast_id,
             colorbar=True,
+            cmap="bwr",
```

Comment:
because it is not a proper diverging colormap 

> Diverging: change in lightness and possibly saturation of two different colors that meet in the middle at an unsaturated color; should be used when the information being plotted has a critical middle value, such as topography or when the data deviates around zero.
>
> https://matplotlib.org/stable/users/explain/colors/colormaps.html#classes-of-colormaps

See also https://matplotlib.org/stable/users/explain/colors/colormaps.html#diverging

I can switch to seismic that has more range

[![image](https://github.com/user-attachments/assets/819c6167-3348-4784-8144-7ace341dd039)](https://matplotlib.org/stable/_images/sphx_glr_colormaps_011.png)